cmd/bindings/bindings.go

```diff
@@ -141,29 +148,61 @@ func cmdBindingsRemove(b *bindings, printf shared.FormatFn) *cobra.Command {
 	return c
 }
 
+func cmdBindingsVerify(b *bindings, printf shared.FormatFn) *cobra.Command {
+	c := &cobra.Command{
+		Use:   "verify [product name (optional)]",
+		Short: "Verify the bound Apigee products and the associated Apps also have remote-service product associated",
+		Long:  "Verify the bound Apigee product (check all bound products if unspecified) and the associated Apps also have remote-service product associated",
+		Args:  cobra.MaximumNArgs(1),
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			apps, err := b.getAppMap()
+			if err != nil {
+				return err
+			}
+			if len(args) == 0 {
+				return b.verifyAll(apps, printf)
+			}
+			p, err := b.getProduct(args[0])
+			if err != nil {
+				return err
+			}
+			b.verify(p, apps, printf)
```

Comment:
Is it possible to exit 0 with no lint errors and exit 1 if there are? Would be convenient for scripting.

Reply:
It is with some loss of code coverage. Let me see how much it changes.

Reply:
The coverage is still fine after this. Exits with 1 on error sounds good.